pom.xml

```diff
@@ -27,6 +27,7 @@
     <lucene.version>3.6.2</lucene.version>
     <powermock.version>1.5.5</powermock.version>
     <logback-classic.version>1.1.2</logback-classic.version>
+    <bouncycastle.version>1.52</bouncycastle.version>
```

Comment:
Release 1.53 indeed worth upgrading to but probably in a separate PR. Here I've just extracted previous version as a parameter.